docs/android/gradle.md

```diff
@@ -1,205 +1,200 @@
 
 # Gradle Build Instructions - Android
 
-These instructions guide you through the process of integrating Product Science instrumentation into an Android application's Gradle build process.
+Follow these steps to instrument your app with PS Plugin. Once you have successfully installed the build and run the app, you will be able to record and analyze traces.
+
+If you get stuck, do refer to our [FAQ](https://docs.productscience.app/android/integration-faq-android/) for additional steps and guidance.
 
 !!! info
     If your build environment does not allow network access to our servers `https://prod.productscience.app/api/v1/*`, please add to your allowlist.
     If your network settings prevent adding this endpoint, you will be provided with plugin and config archives detailed in sections below.
 
-## 1. Credentials
+## 1. Identify the two levels of files
+You will be working with and modifying files at two levels:
+1. The __root__ directory. This is the root directory of your Android project. If your repository is entirely your Android project, it will usually be the top level folder. The `build.gradle` (or `build.gradle.kts`) file at this level is usuall fairly terse, defining a few global dependencies and repositories.
+2. The __app__ directory. This is the folder that actually builds your Android app. It is often, though not always, named `app`. The `build.gradle` file at this level is usually more verbose, containing all the dependencies and settings needed to build your Android app.
```

Comment:
```suggestion
## 1. Identify the two levels of files
## 1. Identify the two levels of files
You will be working with and modifying files at two levels:

1. The __root__ directory. This is the root directory of your Android project. 
If your repository is entirely your Android project, it will usually be the top level folder. 
The `build.gradle` (or `build.gradle.kts`) file at this level is usuall fairly terse, 
defining a few global dependencies and repositories.

2. The __app__ directory. This is the folder that actually builds your Android app. 
It is often, though not always, named `app`. The `build.gradle` file at this level is usually more verbose, 
containing all the dependencies and settings needed to build your Android app.
```

Gradle syntax + line splitting for maintaining